pylint/checkers/classes/class_checker.py

```diff
@@ -730,6 +730,12 @@ def _has_same_layout_slots(
         "Used when we detect that a property also has parameters, which are useless, "
         "given that properties cannot be called with additional arguments.",
     ),
+    "E0245": (
```

Comment:
Would you mind putting this under `E0244`? Just for consistency :)

Reply:
Looks like `E0244` is already taken by `invalid-enum-extension`, this is the next available.